rich/console.py

```diff
@@ -342,6 +343,8 @@ def __exit__(
         exc_tb: Optional[TracebackType],
     ) -> None:
         self._result = self._console.end_capture()
+        if self._echo:
+            self._console.out(self._result)
```

Comment:
This is going to run captured input through `print` again, which may modify the result.

Reply:
right, guess I'll invoke `print` directly then.